modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -115,13 +117,13 @@ CREATE TABLE IDN_OAUTH2_SCOPE (
             NAME VARCHAR2 (255) NULL,
             DESCRIPTION VARCHAR2 (512) NULL,
   TENANT_ID INTEGER DEFAULT 0,
-        ROLES VARCHAR2 (500) NULL,
+	    ROLES VARCHAR2 (500) NULL,
             PRIMARY KEY (SCOPE_ID))
 /
 CREATE SEQUENCE IDN_OAUTH2_SCOPE_SEQUENCE START WITH 1 INCREMENT BY 1 NOCACHE
 /
 CREATE OR REPLACE TRIGGER IDN_OAUTH2_SCOPE_TRIGGER
-            BEFORE INSERT
+		    BEFORE INSERT
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04